build.gradle

```diff
@@ -169,10 +169,10 @@ dependencies {
 //    runtimeOnly fg.deobf("com.teamcofh:thermal_expansion:1.16.5-1.4.2.4")
 
     // create
-    compileOnly fg.deobf("com.jozufozu.flywheel:Flywheel-Forge:${rootProject.flywheel_version}")
-    compileOnly fg.deobf("com.simibubi.create:Create:${rootProject.create_version}")
-//    runtimeOnly fg.deobf("com.jozufozu.flywheel:Flywheel-Forge:${rootProject.flywheel_version}")
-//    runtimeOnly fg.deobf("com.simibubi.create:Create:${rootProject.create_version}")
+    compileOnly fg.deobf("com.jozufozu.flywheel:flywheel-forge-1.18.2:${rootProject.flywheel_version}")
```

Comment:
If you add `{ transitive = false }` you don't need to pull in curios/CC
Reference: https://github.com/Creators-of-Create/Create/wiki/Depending-on-Create